operator/pkg/central/extensions/reconcile_pvc.go

```diff
@@ -225,15 +255,25 @@ func (r *reconcilePVCExtensionRun) getUniqueOwnedPVCs() (*corev1.PersistentVolum
 		return nil, nil
 	}
 
+	// Filter PVC List by current PVC Claim Name
+	filtered := pvcList[:0]
```

Comment:
I will move it a new value with append because the length of pvc list is realistically small, but generally, this is a fine pattern as long as the considerations above are taken into account